Taskfile.yml

```diff
@@ -52,21 +59,20 @@ tasks:
   build-linux:
     platforms: [linux]
     cmds:
-      - cmd: mkdir -p build/wallet-plugin
```

Comment:
Are you sure the `build` directory will be always created ? (same for MacOS and Windows)

Reply:
yes because it's on github and -p option will create the full path, including build directory